src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,75 @@
+import RcMenu, {
+  SubMenu,
+  MenuItem,
+  SubMenuProps,
+  MenuItemProps,
+  MenuProps,
+} from 'rc-menu'
+import { FaChevronRight } from 'react-icons/fa'
+import styles from './Menu.module.scss'
+import csx from 'classnames'
+import { PropsWithChildren } from 'react'
+import 'rc-menu/assets/index.css'
+
+const GroupTitle: React.FC<PropsWithChildren> = ({ children }) => {
+  return <div className="menu-v3__block-title">{children}</div>
+}
+
+const SubMenuCustom: React.FC<SubMenuProps> = ({
+  title,
+  children,
+  className,
+  ...rest
+}) => {
+  return (
+    <SubMenu
+      popupOffset={[8, 0]}
+      className={csx('menu-v3__submenu', className || '')}
```

Comment:
Use css modules here and check it for every other className in this PR that css modules is used.

The same as in the other PR. Use camel-case for class names or even get rid of the scope at all. When using css modules the block part of BEM is already given by the modules scope.
```suggestion
      className={csx(styles.submenu, className || '')}
```